packages/cli/test/integration/generators/controller.integration.js

```diff
@@ -114,6 +114,22 @@ describe('lb4 controller', () => {
       checkRestCrudContents();
     });
 
+    it('creates REST CRUD template with valid input- id required', async () => {
+      await testUtils
+        .executeGenerator(generator)
+        .inDir(SANDBOX_PATH, () =>
+          testUtils.givenLBProject(SANDBOX_PATH, {
+            includeDummyModel: true,
+            includeDummyRepository: true,
+          }),
+        )
+        .withPrompts(
+          Object.assign({}, restCLIInputComplete, {idRequired: true}),
+        );
+
+      checkRestCrudContents(1);
```

Comment:
Boolean arguments are difficult to understand. What does `1` mean in this context? I have to look up the definition of `checkRestCrudContents` to understand that.

Let's use named flags (an `options` arg) please.

```suggestion
      checkRestCrudContents({idRequired: true});
```